src/dxtbx/command_line/radial_average.py

```diff
@@ -351,7 +344,7 @@ def load_func(x):
                     results = results.select(xvals <= params.plot_x_max)
                     xvals = xvals.select(xvals <= params.plot_x_max)
                 if params.x_axis == "resolution":
-                    xvals = 1 / (xvals ** 2)
+                    xvals = 1 / (xvals**2)
```

Comment:
Did you run black manually rather than using pre-commits?

Reply:
Yah that's what I usually do.  I often don't have precommits available so I just run black and flake8 manually.

Reply:
That explains that. You're running the wrong version.

Reply:
I'm using the latest from conda-forge, 22.1.0.  What should I be using?

Reply:
Looks like the [pinned version of black](https://github.com/dials/dials/blob/main/.pre-commit-config.yaml#L11) in the precommittbx isn't available in conda-forge for manual installs.  21.5b2 seems to work tho.